backend/ddang/src/main/java/com/ddang/ddang/authentication/application/AuthenticationService.java

```diff
@@ -0,0 +1,89 @@
+package com.ddang.ddang.authentication.application;
+
+import com.ddang.ddang.authentication.application.dto.TokenDto;
+import com.ddang.ddang.authentication.domain.exception.InvalidTokenException;
+import com.ddang.ddang.authentication.domain.Oauth2UserInformationProviderComposite;
+import com.ddang.ddang.authentication.infrastructure.jwt.PrivateClaims;
+import com.ddang.ddang.authentication.domain.TokenDecoder;
+import com.ddang.ddang.authentication.domain.TokenEncoder;
+import com.ddang.ddang.authentication.domain.TokenType;
+import com.ddang.ddang.authentication.domain.dto.UserInformationDto;
+import com.ddang.ddang.authentication.infrastructure.oauth2.OAuth2UserInformationProvider;
+import com.ddang.ddang.authentication.infrastructure.oauth2.Oauth2Type;
+import com.ddang.ddang.user.domain.User;
+import com.ddang.ddang.user.infrastructure.persistence.JpaUserRepository;
+import java.time.LocalDateTime;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class AuthenticationService {
+
+    private static final String PRIVATE_CLAIMS_KEY = "userId";
+
+    private final Oauth2UserInformationProviderComposite providerComposite;
+    private final JpaUserRepository userRepository;
+    private final TokenEncoder tokenEncoder;
+    private final TokenDecoder tokenDecoder;
+
+    @Transactional
+    public TokenDto login(final Oauth2Type oauth2Type, final String oauth2AccessToken) {
```

Comment:
login보다는 인증 과정을 진행한다는 걸 표현하는게 더 나을 것 같습니다 
매개변수로 굳이 어떤 토큰인지 자세하게 알려줄 필요도 없을 것 같습니다

```suggestion
    public TokenDto authorize(final Oauth2Type oauth2Type, final String accessToken) {
```